firestore/swift/firestore-smoketest/ViewController.swift

```diff
@@ -843,6 +843,20 @@ class ViewController: UIViewController {
         // [END get_multiple_all]
     }
 
+    private func getMultipleAllSubcollection() {
+        // [START get_multiple_all_subcollection]
+        db.collection("cities", "SF", "landmarks").getDocuments() { (querySnapshot, err) in
+            if let err = err {
```

Comment:
@morganchen12 ... comma-delimited in Swift, or slashes?

Reply:
Assuming you're talking about the doc path, slashes universally across platforms.